### ISSUE TITLE
Can't connect to ip address 192.168.44.1:88

### ISSUE THREAD

**Issue by hakankaraduman** [sabhiram/raspberry-wifi-conf#30]
I'm stucked at 
Step 2: Join the above network, and navigate to the static IP and port we set in config.json (http://192.168.44.1:88), you will 
I can't connect it from my pc, but I can open it on rpi. How do I fix this?

Also how do I implement this functionality in an electronjs app, can you give some tips?

**Comment by sabhiram** [comment]
Sorry this took so long to respond to, I have been swamped at work lately.  It seems like the DHCP server is not working?  Can you post the logs from the rpi side of things? 

I meant to debug this - are you on a Pi3?

**Comment by hakankaraduman** [comment]
Yes I'm on pi3 model b. I'm a beginner on linux. Do you want the logs from only the terminal or is there a dump file anywhere else I should look for?

**Comment by mbalgooi** [comment]
I want to use this to setup my Rpi 3B as an wifi AP, is the problem with installing this on a Rpi 3B resolved?

**Comment by sabhiram** [comment]
So I just ran this on my Pi3 and I was able to complete the loop and have the wifi connect to my home wifi. 

Can you try the following:
```
0. In your config file - under "access_point" add a property "force_reconfigure" and set it to "true"
1. Stop the service if you enabled it and `cd` into the project directory
2. Manually run this command `sudo npm start`
3. You should see something like this:

> raspberry-wifi-conf@0.0.1 start /home/pi/work/raspberry-wifi-conf
> node server.js

Wifi is not enabled, Enabling AP for self-configure

Force reconfigure enabled - reset AP
... dhcp server restarted!
ifdown wlan0 successful...
ifup wlan0 successful...
... hostapd restarted!
... AP Enable Success!

HTTP server running...
```

At this point using a different wifi device (I use my android phone), connect to the `rpi-wifi-ap` wifi access point. 

If the above works, you should be able to access `192.168.44.1:88` on a chrome browser.

Please let me know which step fails.  If for you get the "HTTP server running" message but the access point does not show up, let me know (I might have seen that issue once or twice when bringing this up).  Typically restarting the app fixes this hiccup - would love more data on this.
